libdnf/transaction/Swdb.hpp

```diff
@@ -44,7 +44,7 @@ class Transformer;
 
 namespace libdnf {
 
-class Swdb {
+struct Swdb {
 public:
```

Comment:
`public:` is redundant if you use the `struct` to declare, so you can drop it? Otherwise looking good.

Reply:
Yes. The members of struct are public by default.
But I want to be consistent with libdnf code style.